repos/system_upgrade/common/libraries/fetch.py

```diff
@@ -29,12 +29,12 @@ def _request_data(service_path, cert, proxies, timeout=REQUEST_TIMEOUT):
         attempt += 1
         try:
             return requests.get(service_path, cert=cert, proxies=proxies, timeout=REQUEST_TIMEOUT)
-        except requests.exceptions.Timeout as e:
```

Comment:
I don't think that you want to revert these things. That's by mistake I assume

Reply:
++++ Still you need to change this back again.

Reply:
@Rezney

Reply:
fixed...